client/src/assets/scss/_new.scss

```diff
@@ -712,6 +712,7 @@ div.page-people {
         display: flex;
         flex-direction: column;
         height: 100%;
+        overflow-x: hidden;
```

Comment:
What do we solve with this? Do we crop some parts?

Reply:
Well, what happens is that a div that has input inside is few pixels wider that the screen, which case the horizontal overflow to appear.
There's not visual change when it's hidden, so, it's the easiest solution.